autogpt/api_manager.py

```diff
@@ -0,0 +1,161 @@
+from typing import List
+
+import openai
+
+from autogpt.config import Config
+
+cfg = Config()
+openai.api_key = cfg.openai_api_key
+print_total_cost = True
+
+# Define the cost per thousand tokens for each model
+# TODO: make this a json file that we can update separate from the code
+COSTS = {
```

Comment:
Can we extract this to a models.json or a modelsinfo.py. It's duplicated throughout the codebsse and it would be nice to just say use gpt3 and pass that as an object